src/utils/npm.js

```diff
@@ -39,10 +39,24 @@ export function publish(
 ) {
   return npmRequestLimit(async () => {
     logger.info(messages.npmPublish(pkgName));
-    let publishFlags = opts.access ? ['--access', opts.access] : [];
-    return await processes.spawn('npm', ['publish', ...publishFlags], {
-      cwd: opts.cwd
-    });
+    const publishFlags = opts.access ? ['--access', opts.access] : [];
+
+    try {
+      const result = await processes.spawn(
+        'npm',
+        ['publish', ...publishFlags],
+        {
+          cwd: opts.cwd
+        }
+      );
+      return { published: true, publishedPkgInfo: JSON.parse(result.stdout) };
```

Comment:
Not sure about the naming of this one. 

From my understanding, this will return something like

```
$ node-client git:(patch-1) ✗ npm publish
+ @lbatchelor/netlify@1.2.0-patched.6
```

Which wouldnt be parseable.

I would just return true or false from this I think. The error messages should be written to stderr already, so no need for us to capture that.